resources/asciidoctor/bin/asciidoctor

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
Everything below this line is asciidoctor being copied into the repo using bundle. Asciidoctor and Bundler are MIT licensed.

Reply:
...I'm experiencing so many emotions all at once, haha